app/controllers/v1/clusters_controller.rb

```diff
@@ -290,7 +459,7 @@ def deploy
 
     # Getting external files needed by the compose
     externalFiles = []
-    for path in Dir['./mastermind-services/'+serviceTypeName+'/'+serviceTypeVersion+'/*']
+    for path in Dir['./'+serviceType.local_path+'/*']
```

Comment:
this looks like the harvesting of files required to deploy, right? If so, you may want to skip markdown files as well down below